Yoast/javascript
# Add prop to SocialPreviewEditor for replaceVars

## Summary
Add prop to SocialPreviewEditor for replaceVars


This PR can be summarized in the following changelog entry:
* [@yoast/social-metadata-previews] Add a prop to the SocialPreviewEditor for replacement variables.

## Test instructions

This PR can be tested by following these steps:
See https://github.com/Yoast/wordpress-seo/pull/14930

## Quality assurance
